src/lib/gadgets/gadgets.ts

```diff
@@ -314,6 +315,31 @@ const Gadgets = {
     return compactMultiRangeCheck(xy, z);
   },
 
+  /**
+   * Checks that three {@link Field} elements are less than 2^12 (using only one row) by verifying that they exist in the [RANGE_CHECK_TABLE](https://github.com/o1-labs/proof-systems/blob/master/kimchi/src/circuits/lookup/tables/mod.rs).
+   *
+   * It's possible to use this to check that a value is less than x bits by scaling the value by 2^(12 - x) before passing it in (with x < 12).
```

Comment:
as Anais said below, this has to make clearer that for checking fewer bits you have to check _both_ $z$ and $z 2^{12-x}$. 

also, nit: `x` is not the best variable to use for an exponent here because we use it everywhere for field elements (usually in mathematics you try to pick different symbols for different kinds of value). exponent could be $i$ or $n$ or $e$ for example